### PR TITLE
Adapt api.MediaKeySession.onkeystatuseschange to new events structure

### PR DESCRIPTION
**Pull request by queengooborg**
This PR adapts the keystatuseschange event of the MediaKeySession API to conform to the new events structure.

### CODE REVIEW

**Review by foolip — CHANGES_REQUESTED** [review]
https://source.chromium.org/chromium/chromium/src/+/6e3647df59a9b8a2c77d49ce5681e2a5b3ab7d17 shows that the event handler attributes were added after the events themselves. Can you add `partial_implementation` ranges?

**Comment by Elchi3** [comment]
My guess is that this should be:

```
"chrome": [
  {
    "version_added": "55"
  },
  {
    "version_added": "42",
    "version_removed": "55",
    "partial_implementation": true,
    "notes": "The <code>onkeystatuseschange</code> event handler property is not supported."
  }
],
```

**Comment by Elchi3** [comment]
(And similar for the other chromium based browsers)

File: api/MediaKeySession.json [hunk 1]
```diff
@@ -293,47 +293,96 @@
...
           "support": {
-            "chrome": {
-              "version_added": "42"
-            },
-            "chrome_android": {
-              "version_added": "42"
-            },
+            "chrome": [
+              {
+                "version_added": "55"
```
Comment (foolip) [inline api/MediaKeySession.json:304]: Confirmed by https://storage.googleapis.com/chromium-find-releases-static/6e3.html#6e3647df59a9b8a2c77d49ce5681e2a5b3ab7d17

**Review by foolip — APPROVED** [review]

**Pull request merged**